### PR TITLE
closes #60 feat: review reducers, constants, action creators

### LINKED ISSUE #60: Frontend: Review Skeleton

### PR DESCRIPTION
**Pull request by gabrielwr**
closes #60

### CODE REVIEW

File: app/reducers/review.jsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      ))
+      break
+
+    case DELETE_REVIEW:
+      newState.reviews = newState.reviews.filter((currentReview) => (
+        (currentReview.id !== action.reviewId)
+      ))
+      break
+
+    case ADD_REVIEW_TO_ORDER:
```
Comment (gabrielwr) [inline app/reducers/review.jsx]: need to update all references to students. Made a new issue referencing this, disregard it here for now.

File: app/reducers/review.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+/* -----------------    ACTION TYPES     ------------------ */
+const SET_REVIEWS = 'SET_REVIEWS'
+const SET_SELECTED_REVIEW = 'SET_SELECTED_REVIEW'
+const DELETE_REVIEW = 'DELETE_REVIEW'
+const UPDATE_REVIEW = 'UPDATE_REVIEW'
+//do we need a create_review?
+const CREATE_REVIEW = 'CREATE_REVIEW'
+const ADD_REVIEW_TO_ORDER = 'ADD_REVIEW_TO_ORDER'
```
Comment (JeffKandel) [inline app/reducers/review.jsx]: Add Review to Product

File: app/reducers/review.jsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  // Below might not be needed
+  // I am leaving here till we have the order reducer built out.
+const UPDATE_REVIEW_IN_ORDER = 'UPDATE_REVIEW_IN_ORDER'
```
Comment (JeffKandel) [inline app/reducers/review.jsx]: Update Review in Product

File: app/reducers/review.jsx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+const REMOVE_REVIEW_FROM_ORDER = 'REMOVE_REVIEW_FROM_ORDER'
```
Comment (JeffKandel) [inline app/reducers/review.jsx]: remove REview from Product

**Review by JeffKandel — CHANGES_REQUESTED** [review]
Clean up naming here. see Below for suggestions

**Pull request merged**